api/src/test/java/org/openmrs/api/UserServiceTest.java

```diff
@@ -1650,4 +1650,12 @@ private void withCurrentUserAs(User user, Runnable internals) throws IllegalAcce
 			FieldUtils.getField(UserContext.class, "user", true).set(userContext, authenticatedUser);
 		}
 	}
+
+	@Test
+	public void saveUserProperty_shouldAddANewPropertyWithAVeryLargeStringWithoutRunningIntoError() {
+		final String USER_PROPERTY_KEY = "emrapi.lastViewedPatientIds";
```

Comment:
Since you are adjusting both the key and value size, can you reflect both in the test?